docs/snapshotters/erofs.md

```diff
@@ -128,6 +128,44 @@ $ # run the container with the provides snapshotter
 $ ctr run -rm -t --snapshotter erofs docker.io/library/busybox:latest hello sh
 ```
 
+## Data Integrity
+
+The EROFS snapshotter provides two methods to consolidate data integrity:
+
+### Data Integrity with Immutable File Attribute
+
+By setting `set_immutable = true`, the EROFS snapshotter marks each layer blob
+with `IMMUTABLE_FL`. This ensures that dirty data is flushed immediately and the
+EROFS layer blob cannot be deleted, renamed, or modified.
+
+Since it can flush in-memory dirty data, it may greatly impact container launch
+performance.
+
+### Data Integrity with fs-verity
+
+By setting `enable_fsverity = true`, the EROFS snapshotter will:
+
+ - Enable fs-verity on EROFS layers during commit;
+
+ - Verify the fs-verity status before mounting layers;
+
+ - Skip fs-verity if the filesystem or kernel does not support it.
+
+The fs-verity feature strictly ensures data integrity for container image
+filesystems.
+
+### Pros and Cons
+
+Both methods help ensure data integrity and can be enabled together:
+
+ - The immutable file attribute is mainly used to ensure data persistence and
```

Comment:
I think this is better if its in the section with the option.

Reply:
done.